src/utils/frame.js

```diff
@@ -47,6 +47,14 @@ function isExtJs(frame) {
   return /\/ext-all[\.\-]/.test(getFrameUrl(frame));
 }
 
+function isUnderscore(frame) {
+  return getFrameUrl(frame).match(/underscore/i);
+}
+
+function isLodash(frame) {
+  return getFrameUrl(frame).match(/lodash/i);
+}
+
```

Comment:
For the regexp I'm not sure how loose you'd prefer but you might tighten it up to `/\blodash\b/`